README.md

```diff
@@ -33,10 +77,12 @@ name: fossa
 steps:
 - name: fossa  
   image: rancher/drone-fossa:latest
+  failure: ignore  
   settings:
     api_key:
       from_secret: FOSSA_API_KEY
-    command: test
+    commands: 
+      - test
```

Comment:
I think this is the only relevant portion of code for this repository.

Reply:
Either syntax could work:

```
command: test
```

-Or-

```
commands:
  - test
```

Here is the example of the first: https://drone-publish.rancher.io/rancherlabs/drone-plugin-fossa/15/1/3